packages/portal/src/lang/en.js

```diff
@@ -1099,6 +1101,10 @@ export default {
     },
     "results": {
       "limitWarning": "Additional results are not shown as only the first 1000 most relevant results are shown. If you haven't found what you're looking for, please consider refining your search.",
+      "login": "Do not translate",
+      "loginToSeeMore": "{login} Do not translate",
+      "loginToSeeMultilingualResults": "Log in to search across the entire collection, and get results not limited to the language of your search term.",
+      "showingMultilungualResults": "Showing results across the entire collection and all languages.",
```

Comment:
This should be `showingMultilingualResults`

Reply:
Updated.